docs/TROUBLESHOOT.md

```diff
@@ -0,0 +1,55 @@
+# Wireless Meshing
+
+Below are a few things you should look at when diagnosing connections. These can help identify bad configurations or other errors.
+
+## ISBB
```

Comment:
```suggestion
## IBSS
```

Should this document explain what IBSS is to some degree, so people know whether to run these commands?